FileProcessor/src/main/java/elements/Split.java

```diff
@@ -24,8 +30,41 @@ public Split() {
 
     @Override
     public void process() {
-        throw new UnsupportedOperationException("Not supported yet."); // Generated from nbfs://nbhost/SystemFileSystem/Templates/Classes/Code/GeneratedMethodBody
-    }
+        ArrayList<Entry> entries = getInputEntries();
+        ArrayList<Entry> output = new ArrayList<Entry>();
+        String contents;
+        String[] contentsByLine;
+        String temp;
+        int num=-1;
+
+        for (int i = 0; i < entries.size(); i++) {
+            try {
+                Entry entry = entries.get(i);
+                contents = entry.getContents();
+                contentsByLine = contents.split("\n");
+                
+                for (int j = 0; j < contentsByLine.length;) {
+                    
+                    num++;
+                    temp = "";
+                    Entry clone = entry.clone();
+                    for (int count = lines; count > 0 && j < contentsByLine.length; count--, j++) {
+                        temp+=contentsByLine[j]+"\n";
+                    }
+                    clone.setAddress(entry.getAddress().split("\\.")[0]+num+".txt");
```

Comment:
idk if theses files are always going to be .txt files? What if we get some other files?